tests/unittest_builder.py

```diff
@@ -70,12 +70,12 @@ def test_callfunc_lineno(self) -> None:
         self.assertEqual(name.tolineno, 4)
         strarg = callfunc.args[0]
         self.assertIsInstance(strarg, nodes.Const)
-        if hasattr(sys, "pypy_version_info"):
-            lineno = 4
+        if hasattr(sys, "pypy_version_info") or not PY38_PLUS:
```

Comment:
I believe `PY38_PLUS` also works for `PyPy`. Could you check this? The old `hasattr` might be unnecessary.